src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -27,6 +27,13 @@
            vals
            (sort-by :position)))
 
+(rf/reg-sub
+ :wallet/other-accounts
+ :<- [:wallet/accounts]
+ :<- [:wallet/current-viewing-account-address]
+ (fn [[accounts current-address]]
+   (filter #(not= (:address %) current-address) accounts)))
+
```

Comment:
There is an existing sub `:wallet/accounts-without-current-viewing-account` which gives out the same result.

Reply:
Thanks I'll use that instead